sdks/go/pkg/beam/testing/passert/equals_test.go

```diff
@@ -130,3 +131,64 @@ func TestEqualsList_Bad(t *testing.T) {
 		}
 	}
 }
+
+func ExampleEquals() {
+	p, s := beam.NewPipelineWithRoot()
+	col := beam.Create(s, "some", "example", "strings")
+
+	Equals(s, col, "example", "some", "strings")
+	err := ptest.Run(p)
+	fmt.Println(err == nil)
+
+	// Output: true
+}
+
+func ExampleEquals_pcollection() {
+	p, s := beam.NewPipelineWithRoot()
+	col := beam.Create(s, "some", "example", "strings")
+	exp := beam.Create(s, "example", "some", "strings")
+
+	Equals(s, col, exp)
+	err := ptest.Run(p)
+	fmt.Println(err == nil)
+
+	// Output: true
+}
+
+func ExampleEqualsList() {
+	p, s := beam.NewPipelineWithRoot()
+	col := beam.Create(s, "example", "inputs", "here")
+	list := [3]string{"here", "example", "inputs"}
+
+	EqualsList(s, col, list)
+	err := ptest.Run(p)
+	fmt.Println(err == nil)
+
+	// Output: true
+}
+
+func ExampleEqualsList_mismatch() {
+	p, s := beam.NewPipelineWithRoot()
+	col := beam.Create(s, "example", "inputs", "here")
+	list := [3]string{"wrong", "inputs", "here"}
+
+	EqualsList(s, col, list)
+	err := ptest.Run(p)
+	if wrapper, ok := err.(interface{ Unwrap() error }); ok {
+		err = wrapper.Unwrap()
+	}
```

Comment:
Consider hiding this in a helper method.  

`err = unwrapError(err)`

Unnecessary details can detract from an example's utility.

Reply:
Change made.